### PR TITLE
New Sectors! Coalition Focused

### PR DESCRIPTION
**Pull request by Caelphon**
This adds the Haneunim Sector, or more so the Konyanger Solar System. Majority of the assets are holdovers until additional development has taken place. This will remain WIP until I do some dungeons with Readthisnameplz (Mini Moose). Also adds Burzsia and Weeping Stars.

### CODE REVIEW

**Comment by Caelphon** [comment]
!wip

**Comment by Caelphon** [comment]
!lore

File: maps/away/ships/tcfl_patrol/tcfl_peacekeeper_ship.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	name = "TCFL Corvette"
 	description = "Serving as the very foundation of the SCC's (And more specifically, NanoTrasen's) fleet of asset protection vessels, the Cetus-class is versatile and durable, but also clumsy and somewhat underpowered in regards to its engine and propulsion. It features small weapon hardpoints in its thruster arms, and a massive hangar host to the design's interdiction counterpart - the Hydrus-class shuttle. This one appears to be a Decanus-class, the Tau Ceti Foreign Legion variation of the design."
 	suffixes = list("ships/tcfl_patrol/tcfl_peacekeeper_ship.dmm")
-	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_BADLANDS)
+	sectors = list(SECTOR_TAU_CETI, SECTOR_ROMANOVICH, SECTOR_CORP_ZONE, SECTOR_VALLEY_HALE, SECTOR_BADLANDS, SECTOR_WEEPING_STARS, SECTOR_BURZSIA)
```
Comment (TheDanseMacabre) [inline maps/away/ships/tcfl_patrol/tcfl_peacekeeper_ship.dm:5]: I really don't see why the TCFL would be active in Coalition space.

File: code/modules/maps/planet_types/lore/haneunim.dm [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+/obj/effect/overmap/visitable/sector/exoplanet/grass/grove/konyang
+	name = "Konyang"
+	desc = "The jewel of Haneunim, lush with life."
+	color = "#346934"
+	planetary_area = /area/exoplanet/grass/grove
+	rock_colors = list(COLOR_BEIGE, COLOR_PALE_YELLOW, COLOR_GRAY80, COLOR_BROWN)
+	grass_color = null
+	plant_colors = null
+	map_generators = list(/datum/random_map/noise/exoplanet/grass/grove)
+	possible_themes = list(/datum/exoplanet_theme/mountains/breathable)
```
Comment (WickedCybs) [inline code/modules/maps/planet_types/lore/haneunim.dm:105]: Remove the mountain theme from this.

File: code/modules/maps/planet_types/snow.dm [hunk 3]
```diff
@@ -6,6 +6,7 @@
 	rock_colors = list(COLOR_DARK_BLUE_GRAY, COLOR_GUNMETAL, COLOR_GRAY80, COLOR_DARK_GRAY)
 	plant_colors = list("#d0fef5","#93e1d8","#93e1d8", "#b2abbf", "#3590f3", "#4b4e6d")
 	map_generators = list(/datum/random_map/noise/exoplanet/snow)
+	possible_themes = list(/datum/exoplanet_theme/mountains/breathable)
```
Comment (WickedCybs) [inline code/modules/maps/planet_types/snow.dm:9]: Remove this, it's not needed for generic snow planets should we have them.

**Review by WickedCybs — CHANGES_REQUESTED** [review]
As per my comments, remove the mountain theme for Konyang and remove that addition from the snow exoplanet. 

Danse is right that the TCFL does not belong in the sectors you added, so remove those additions too.

File: code/modules/background/space_sectors/coalition.dm [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+	possible_exoplanets = list(/obj/effect/overmap/visitable/sector/exoplanet/lava/huozhu, /obj/effect/overmap/visitable/sector/exoplanet/barren/hwanung, /obj/effect/overmap/visitable/sector/exoplanet/barren/qixi, /obj/effect/overmap/visitable/sector/exoplanet/grass/grove/konyang)
+	starlight_color = "#b13636"
+	starlight_power = 2
+	starlight_range = 4
+
+/datum/space_sector/weeping_stars
+	name = SECTOR_WEEPING_STARS
+	description = "The wounds of the Great Interstellar War still have not healed, with planets dotting the Weeping Stars bearing their scars from the massive conflict."
+	skybox_icon = "badlands"
+	possible_exoplanets = list(/obj/effect/overmap/visitable/sector/exoplanet/barren/asteroid, /obj/effect/overmap/visitable/sector/exoplanet/lava, /obj/effect/overmap/visitable/sector/exoplanet/barren, /obj/effect/overmap/visitable/sector/exoplanet/snow, /obj/effect/overmap/visitable/sector/exoplanet/grass/grove, /obj/effect/overmap/visitable/sector/exoplanet/desert)
```
Comment (WickedCybs) [inline code/modules/background/space_sectors/coalition.dm:14]: I don't really want the snow, grove and desert exoplanets spawning at all until there's actual content for them, so I'll be seeing with the others if this should be excluded from the list here.

File: aurorastation.dme [hunk 5]
```diff
@@ -3369,10 +3374,6 @@
 #include "maps\random_ruins\exoplanets\raskara\raskara_ritual.dm"
 #include "maps\random_ruins\exoplanets\raskara\raskara_wreck.dm"
 #include "maps\random_ruins\space_ruins\space_ruins.dm"
-#include "maps\runtime\runtime_overmap.dm"
```
Comment (WickedCybs) [inline aurorastation.dme:3372]: Bring these includes back. Dunno how Reads thing removed those.

File: maps/away/ships/tcfl_patrol/tcfl_peacekeeper_ship.dm (hunk 1 above)
Comment (Caelphon) [reply maps/away/ships/tcfl_patrol/tcfl_peacekeeper_ship.dm:5]: Specifically, the Weeping Stars region is said to be patrolled by both the TCFL and Coalition Rangers to protect trade routes passing through the area.

**Comment by NonQueueingMatt** [comment]
reopen when ready

**Pull request closed without merging**